app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -501,6 +502,23 @@ public boolean onKey(View v, int keyCode, KeyEvent event) {
                     }
                 }
 
+                // Control fast forward and rewind if overlay hidden and not showing live TV
+                if (!mPlaybackController.isLiveTv()) {
+                    if (keyCode == KeyEvent.KEYCODE_MEDIA_FAST_FORWARD || keyCode == KeyEvent.KEYCODE_BUTTON_R1 || keyCode ==KeyEvent.KEYCODE_BUTTON_R2) {
+                        mPlaybackController.skip(30000);
+                        if (!mIsVisible) show();
+                        setFadingEnabled(true);
+                        return true;
+                    }
+
+                    if (keyCode == KeyEvent.KEYCODE_MEDIA_REWIND || keyCode == KeyEvent.KEYCODE_BUTTON_L1 || keyCode == KeyEvent.KEYCODE_BUTTON_L2) {
+                        mPlaybackController.skip(-11000);
+                        if (!mIsVisible) show();
+                        setFadingEnabled(true);
+                        return true;
+                    }
+                }
+
                 if (!mIsVisible) {
```

Comment:
I expect everything inside this statement to probably trigger twice on non-firetv devices

Reply:
Will be good to get a test to see, and if so I can add in the fire TV check.

Reply:
Testing in Android TV emulator in AVD has the expected behaviour using the emulated d-pad fast forward and rewind controls.

Reply:
I did some testing and it, indeed, works as expected! 🎉